src/Paysera/WalletApi/Mapper.php

```diff
@@ -1221,6 +1221,9 @@ protected function decodeAccount($data)
         $account = new Paysera_WalletApi_Entity_Wallet_Account();
 
         $this->setProperty($account, 'number', $data['number']);
+        if (isset($data['owner_type'])) {
+            $this->setProperty($account, 'ownerType', $data['owner_type']);
```

Comment:
I'd use explicit setter to set property, so it's trivial to find usages with IDEA

Reply:
I'm just pointing out that copy-paste approach is not always the best one.